rubocop.yml

```diff
@@ -173,3 +173,29 @@ Layout/FirstHashElementIndentation:
   # This one in particular forbids what Emacs does by default for multi-line
   # hash entries
   Enabled: false
+
+# Pending Cops as of 0.87.1
```

Comment:
This file is getting long. Some day we'll need to organize it more carefully. This section is expected to be removed as of the next version bump when the pending cops are all enabled.